Tools/clinic/clinic.py

```diff
@@ -1867,7 +1866,10 @@ def print_block(self, block, *, core_includes=False):
                 output += '\n'
             write(output)
 
-        arguments="output={} input={}".format(compute_checksum(output, 16), compute_checksum(input, 16))
+        arguments="output={output} input={input}".format(
```

Comment:
Since we're touching this line anyways, let's align it with the existing style.

```suggestion
        arguments = "output={output} input={input}".format(
```